packages/datadog-plugin-jest/src/util.js

```diff
@@ -47,10 +50,46 @@ function getJestTestName (test) {
   return titles.join(' ')
 }
 
+function isMarkedAsUnskippable (test) {
+  let docblocks
+
+  try {
+    const testSource = readFileSync(test.path, 'utf8')
+    docblocks = parse(extract(testSource))
+  } catch (e) {
+    // If we have issues parsing the file, we'll assume no unskippable was passed
+    return false
+  }
+
+  // docblocks were correctly parsed by it does not include a @datadog block
+  if (!docblocks?.datadog) {
+    return false
+  }
+
+  try {
+    return JSON.parse(docblocks.datadog).unskippable
+  } catch (e) {
+    // If the @datadog block comment is present but malformed, we'll run the suite
+    return true
```

Comment:
Why is this true?
if customer is doing some test, and he does something like:
```
/**
 * @datadog NOPE
 */
```
It will considered like unskippable true

Reply:
yeah the rationale is the following: if a customer has _tried_ to add a `@datadog` comment has failed to do it correctly, the most likely reason is that they want this suite to be unskippable (that is, they want this suite to always run, which is the safest fallback). If they don't care whether this suite runs or not (that is, they leave this decision to ITR), most likely they'll just add _nothing_ in this file

Reply:
to put it in another way: it'd be unlikely for a customer to _add_ `@datadog {"unskippable": false}`.